bin.src/report.py

```diff
@@ -244,63 +277,69 @@ def printSummary(dbm, entries):
     avg = executionsPerSlot.average()
     min = executionsPerSlot.min()
     max = executionsPerSlot.max()
-    print "Minimum executions per slot: %d" % min
-    print "Maximum executions per slot: %d" % max
-    print "Mean executions per slot: %d" % avg
-    print
+    print("Minimum executions per slot: %d" % min)
+    print("Maximum executions per slot: %d" % max)
+    print("Mean executions per slot: %d" % avg)
+    print()
 
     newJobStart = NewJobStart(dbm)
     totals = newJobStart.calculate()
 
-    
     # execution switch over
     if len(list(totals)) == 0:
-        print "Could not calculate execution times between workers because"
-        print "no valid entries were found.  Check to see if workers were"
-        print "executed, and/or if valid slot names were set."
-        print
+        print("Could not calculate execution times between workers because")
+        print("no valid entries were found.  Check to see if workers were")
+        print("executed, and/or if valid slot names were set.")
+        print()
     else:
-        print "Time from the end of one worker until the next worker starts"
+        print("Time from the end of one worker until the next worker starts")
         totalStarts = 0
         totalMinutes = 0
-        for key,value in totals.iteritems():
+        for key, value in totals.items():
             if key == -1:
-                print "Single worker started: %d worker%s total" % (value, 's' if value > 1 else '')
+                print("Single worker started: %d worker%s total" % (value, 's' if value > 1 else ''))
             else:
-                print "%d second%s until next worker started: %d worker%s total" % (key, 's' if key > 1 else '', value, 's' if value > 1 else '')
+                pS = 's' if key > 1 else ''
+                pW = 's' if value > 1 else ''
+                print("%d second%s until next worker started: %d worker%s total" % (key, pS, value, pW))
                 totalMinutes = totalMinutes + key*value
                 totalStarts = totalStarts + value
         if totalStarts == 0:
-            print "No workers scheduled for more than one slot"
+            print("No workers scheduled for more than one slot")
         else:
-            print "Mean time to next worker start: %.2f seconds" % (totalMinutes/float(totalStarts))
+            print("Mean time to next worker start: %.2f seconds" % (totalMinutes/float(totalStarts)))
```

Comment:
It might be cleaner to import `division` from `__future__` and remove the `float` here.